web/onos-gui/package.json

```diff
@@ -24,7 +24,7 @@
     "d3": "~5.15.0",
     "google-protobuf": "~3.11.4",
     "grpc-web": "~1.0.7",
-    "gui2-fw-lib": "^2.3.3",
+    "gui2-fw-lib": "^2.3.5",
```

Comment:
this worries me a bit - you changed this in the last patch - did you checkout master and do `gp` after the last change went in?